accelerator/server.cc

```diff
@@ -106,6 +106,39 @@ int main(int argc, char* argv[]) {
     logger_helper_destroy();
   }
 
+  /**
+   * @method {get} / Show the information about a running accelerator
+   *
+   * @return {String[]} object Info of a running accelerator
+   */
+  mux.handle("/")
+      .method(served::method::OPTIONS,
+              [&](served::response& res, const served::request& req) { set_method_header(res, HTTP_METHOD_OPTIONS); })
+      .get([&](served::response& res, const served::request& req) {
+        status_t ret = SC_OK;
+        char* json_result = NULL;
+
+        cJSON* json_obj = cJSON_CreateObject();
```

Comment:
I prefer the current fasion, that an api_* function is called in `server.cc` and in that function a serialization function is called for serialization the response

Reply:
The way you did in the PR is kind not uniform in coding style.